MvvmCross/Windows/Uwp/Views/MvxWindowsViewPresenter.cs

```diff
@@ -29,11 +29,18 @@ public override void Show(MvxViewModelRequest request)
         {
             try
             {
-                var requestTranslator = Mvx.Resolve<IMvxViewsContainer>();
-                var viewType = requestTranslator.GetViewType(request.ViewModelType);
-
-                var converter = Mvx.Resolve<IMvxNavigationSerializer>();
-                var requestText = converter.Serializer.SerializeObject(request);
+                var requestTranslator = Mvx.Resolve<IMvxWindowsViewModelRequestTranslator>();
+                string requestText = string.Empty;
+                if (request is MvxViewModelInstanceRequest)
```

Comment:
can be written `if (request is MvxViewModelInstanceRequest instanceRequest)` and cast below removed.